Suko/View Controller/SUKHomeViewController.m

```diff
@@ -80,11 +81,11 @@ - (void)segueSUKHomeTableViewCell:(SUKHomeTableViewCell *) cell {
     NSMutableDictionary *senderDict = [[NSMutableDictionary alloc] init];
```

Comment:
Can use the pattern `[Class new]` to instantiate objects instead of `[[Class alloc] init]`, which can be less cumbersome. Definitely don't need to change this though, more of just an fyi.